developers_chamber/ecs_utils.py

```diff
@@ -449,6 +450,29 @@ def get_tasks_for_service(cluster, service, region):
     return tasks_arns
 
 
+def stop_services_and_wait_for_tasks_to_stop(cluster, services, timeout, region):
+    ecs_client = _get_ecs_client(region)
+
+    THREAD_MAX = 32
+    number_of_services = len(services)
+
+    tasks = []
```

Comment:
running_tasks? waiting_tasks?

Reply:
jen bych to nějak lépe přejmenoval, když jsem to četl tak mě to trošku zblblo..

možná running_service_tasks

Reply:
chápu, tak ta funkce se jmenuje stop services and wait for tasks to stop, takže jsem si říkal že je jasné o jaké tasky jde. ale přejmenuju

Reply:
upraveno